src/DeviceInput/InputDevices/deviceEnums.ts

```diff
@@ -0,0 +1,189 @@
+/**
+ * Enum for Device Types
+ */
+export enum DeviceType {
+    /** Keyboard */
+    Keyboard = 0,
+    /** Mouse */
+    Mouse = 1,
+    /** Touch Pointers */
+    Touch = 2,
+    /** PS4 Dual Shock */
+    DualShock = 3,
+    /** Xbox */
+    Xbox = 4,
+    /** Switch Controller */
+    Switch = 5,
+    /** Generic Device */
+    GenericDevice = 6
+}
+
+// Device Enums
+/**
+ * Enum for All Pointers (Touch/Mouse)
+ */
+export enum PointerInputs
+{
+    /** Horizontal Axis */

```

Comment:
What drives this specific pointer input enum?

Reply:
This enum is used with any Touch or Mouse input.  Touch inputs use the first three values while Mouse inputs use all of them.